test/e2e/tests/02-blank-form-fails.js

```diff
@@ -0,0 +1,88 @@
+const request = require('request');
+
+const report = require('../report');
+const config = require('../../../config');
+
+// FIXME: This should come in from a config variable
+const url = 'http://localhost:' + config.port + '/healthcare/apply/application/review-and-submit'; // eslint-disable-line
+
+// Test timeout constants
+// TODO(awong): Move to a common file usable for all tests.
+const timeouts = {
+  normal: 500,     // The normal timeout to use. For most opreations w/o a server roundtrip, this should be more than fast enough.
+  slow: 1000,      // A slow timeout incase the page is doing something complex.
+  molasses: 5000,  // A really really slow timeout. This should rarely be used.
+  submission: 10000 // Only to be used for submission.
+};
+
+// TODO(awong): Move this into a custom command or assertion that can be used with client.expect.element().
+function expectNavigateAwayFrom(client, urlSubstring) {
+  client.expect.element('.js-test-location').attribute('data-location')
+    .to.not.contain(urlSubstring).before(timeouts.normal);
+}
+
+const section = [
```

Comment:
Probably... I copied from elsewhere and left the plurality as original. I never read code as english anymore b/c I'm old and seen too much code where the names are short or nonsensical.